bundler/lib/bundler/checksum.rb

```diff
@@ -217,18 +222,27 @@ def to_lock(spec)
       private
 
       def register_checksum(lock_name, checksum)
-        return unless checksum
```

Comment:
You realized we were always passing non nil here, correct?

Reply:
Yes, this was always guarded by the callers, and the method is private.